homeassistant/components/awair/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Awair",
+        "description": "You must register for an Awair developer access token at: https://developer.getawair.com/onboard/login",
+        "data": {
+          "access_token": "Awair access token"
+        }
+      }
+    },
+    "error": {
+      "auth": "Invalid access token.",
+      "unknown": "Unknown Awair API error."
```

Comment:
Please use as many common keys as possible. Here are the available common keys:
https://github.com/home-assistant/core/blob/3ae4eba457160b22bf53fbc6383705c6947efefe/homeassistant/strings.json#L19-L57

Here's an example:
https://github.com/home-assistant/core/blob/3ae4eba457160b22bf53fbc6383705c6947efefe/homeassistant/components/adguard/strings.json#L7

Reply:
Thank you; this is helpful and I'll update.

Reply:
I've updated to use the common keys. I noticed that the translation script doesn't seem to substitute them into the `translations/en.json` file, but I presume that's something that gets fixed in the build process w/ lokalise?

Reply:
Yes. :+1: